includes/Pages/Statistics/StatsMain.php

```diff
@@ -75,20 +75,22 @@ private function generateSmallStatsTable()
         $unconfirmedStatement->closeCursor();
         $this->assign('statsUnconfirmed', $unconfirmed);
 
-        $userStatusStatement = $database->prepare('SELECT COUNT(*) FROM user WHERE status = :status;');
+        $userRoleStatement = $database->prepare('SELECT COUNT(*) FROM user INNER JOIN userrole ON user.id = userrole.user WHERE role = :role;');
```

Comment:
This will include all users with those roles, whether they're active or suspended. Is this what we _want_ to show? If we only want to show active users, we'll need a where condition on user.status == Active too

Reply:
I think we want to only show active users. Pushed a new commit, please take a look.